dd-sdk-android/src/main/kotlin/com/datadog/android/DatadogConfig.kt

```diff
@@ -34,16 +34,16 @@ private constructor(
         val clientToken: String,
         val applicationId: UUID,
         val endpointUrl: String,
-        val serviceName: String,
-        val envName: String
+        val envName: String,
+        val serviceName: String? = null
```

Comment:
Please avoid changing parameters order, this is a nightmare whenever we have a rebase/merger afterwards. 
Also you don't need a default value here, the default will come from the config builder